<p><link rel="stylesheet" href="https://cdn.simplecss.org/simple.min.css"></p>
<p><link rel="stylesheet" href="custom.css"></p>
<h3 id="parker-knight">Parker Knight</h3>
<p>I am a doctoral student in biostatistics at Harvard University, supported by an NSF Graduate Research Fellowship.</p>
<p>My research focuses on high-dimensional estimation, prediction, and signal detection problems in statistical genetics. Recently, I’m thinking about:</p>
<ul>
<li>How can we leverage genetic correlations between traits to improve polygenic risk prediction? With what confidence can we make these predictions when we only have access to GWAS summary statistics?</li>
<li>What are the fundamental limits of testing for multi-phenotype-multi-genotype assocations under structured alternatives? Can we efficiently scale these tests to large, high-dimensional datasets?</li>
</ul>
<p>I work closely on these problems with <a href="https://www.hsph.harvard.edu/profile/rui-duan/">Rui Duan</a> and <a href="https://scholar.harvard.edu/rajarshi/home">Rajarshi Mukherjee</a>.</p>
<p>Before coming to Harvard, I was a mathematics major at the University of Florida, where I explored interests in neuroscience, bioinformatics, and functional programming. I spent a summer working with <a href="https://www.fredhutch.org/en/faculty-lab-directory/randolph-tim.html">Tim Randolph</a> at the Fred Hutch in Seattle, WA, and served on the executive board of UF’s Center for Undergraduate Research.</p>
<p>You can find me on <a href="https://www.github.com/pknight24">GitHub</a> and <a href="https://scholar.google.com/citations?user=NRV4UhwAAAAJ&amp;hl=en&amp;oi=ao">Google Scholar</a>.</p>
